api/src/modules/characters/controllers/character.ctrl.spec.ts

```diff
@@ -0,0 +1,115 @@
+import assert from 'assert';
+
+import status from 'http-status-codes';
+
+import testApi from '../../../shared/spec/api';
+import { createTestUserAndLogin } from '../../../shared/spec/db';
+import {
+  createPostTestForGuardAgainstUnlogin,
+  createGetTestForGuardAgainstUnlogin,
+} from '../../../shared/spec/security';
+
+type createCharReqType = {
+  name: string;
+  race: string;
+};
+
+function createCharacter(
+  token: string,
+  creationPayload: createCharReqType = {
+    name: 'Some name',
+    race: 'elf',
+  }
+) {
+  return testApi(token)
+    .post('/users/current/characters')
+    .send(creationPayload);
+}
+
+describe('Character controller', () => {
+  describe('create new character', () => {
+    it('should not create character with not logged in user', async () => {
+      const createCharReq = {
+        name: 'Some name',
+        race: 'elf',
+      };
+
+      await createPostTestForGuardAgainstUnlogin(
+        '/users/current/characters',
+        createCharReq
+      );
+    });
+    it('should create character with logged in user', async () => {
+      const token = await createTestUserAndLogin();
+
+      const createCharReq: createCharReqType = {
+        name: 'Some name',
+        race: 'elf',
+      };
+
+      const { body } = await createCharacter(token, createCharReq).expect(
+        status.CREATED
+      );
+
+      assert.strictEqual(body.name, createCharReq.name);
+      assert.strictEqual(body.race.key, createCharReq.race);
+    });
+  });
+  describe('find one', () => {
+    it('should not find one with not logged in user', async () => {
+      const token = await createTestUserAndLogin();
+      const resCreation = await createCharacter(token);
```

Comment:
please, always test  response status code. without it it can be harder to decode what's going on when the test fail.